src/components/v2v3/V2V3Project/V2V3ProjectSettings/pages/VerifyTwitterSettingsPage/VerifyTwitterSettingsPage.tsx

```diff
@@ -0,0 +1,121 @@
+import { t, Trans } from '@lingui/macro'
+import { Button } from 'antd'
+import axios from 'axios'
+import { ProjectMetadataContext } from 'contexts/projectMetadataContext'
+import { ThemeContext } from 'contexts/themeContext'
+import { V2V3ProjectContext } from 'contexts/v2v3/V2V3ProjectContext'
+import { useTwitterAuth } from 'hooks/SocialAuth'
+import useTwitterVerified from 'hooks/TwitterVerified'
+import { useWallet } from 'hooks/Wallet'
+import { useContext, useState } from 'react'
+import {
+  emitErrorNotification,
+  emitSuccessNotification,
+} from 'utils/notifications'
+import { verifyWallet } from 'utils/signature'
+
+export const VerifyTwitterSettingsPage = () => {
+  const { projectOwnerAddress } = useContext(V2V3ProjectContext)
+  const { signer, userAddress } = useWallet()
+  const { projectId, projectMetadata, cv } = useContext(ProjectMetadataContext)
+  const { onOpen } = useTwitterAuth()
+  const [isDisconnecting, setIsDisconnecting] = useState(false)
+  const [isAuthenticating, setIsAuthenticating] = useState(false)
+  const { colors } = useContext(ThemeContext).theme
+  const verification = useTwitterVerified()
+  const isOwner =
+    projectOwnerAddress?.toLowerCase() === userAddress?.toLowerCase()
+
+  const initiateAuthRequest = async () => {
+    if (!projectId || !projectMetadata?.twitter) return
+    setIsAuthenticating(true)
+    try {
+      await axios.post(`/api/auth/twitter`, {
+        projectId,
+        cv,
+      })
+      onOpen()
+    } catch (error) {
+      emitErrorNotification(t`Something went wrong.`)
```

Comment:
```suggestion
      emitErrorNotification(t`Twitter authentication failed.`)
```

+, is there some guidance we can give the user here?